### DISCUSSION TITLE
How to only allow a subset of Markdown to be converted to HTML?

### DISCUSSION THREAD

**Discussion by dustinmoris** [yuin/goldmark#320] [Q&A]
Is it possible to configure goldmark so that only a subset of Markdown elements get converted to HTML? For instance if I only configure that bold text and lists should work then this input:

```
# Test

This is a **list**:

- A
- B
- C
```

Then I'd expect this output:

```
# Test

This is a <b>list</b>:

<ul><li>A</li><li>B</li><li>C</li></ul>
```

Is that possible?

**Comment by rhysd** [comment]
That's not possible.

Workaround I can come up is:

1. Parse entire input using `goldmark.Parser.Parse`.
2. Find the subtree node you want to convert using `goldmark/ast.Walk`
3. Get the range (start/end byte offset) of the node
4. Get subslice of the entire input with the range
5. Convert the subslice into HTML using `goldmark.Markdown.Convert`
6. Replace the subslice with the converted text
